packages/p2p/src/components/modal-manager/modals/recommended-modal/recommended-modal.tsx

```diff
@@ -15,14 +18,10 @@ const RecommendedModal = ({ message }) => {
                 </Text>
             </Modal.Body>
             <Modal.Footer>
-                <Button large primary onClick={hideModal} text={localize('Ok')} />
+                <Button large primary onClick={() => hideModal()} text={localize('Ok')} />
```

Comment:
whats the reason behind this?

Reply:
we get this eslint issue if we don't do it this way when passing hideModal to onClick

this is the error
`Type '(options?: THideModalOptions | undefined) => void' is not assignable to type 'MouseEventHandler<HTMLButtonElement>'.
  Types of parameters 'options' and 'event' are incompatible.
    Type 'MouseEvent<HTMLButtonElement, MouseEvent>' has no properties in common with type 'THideModalOptions'.`